### PR TITLE
r-geoaxe, r-oai, r-rbison, r-rebird, r-ridigbio, r-rvertnet, r-wicket

### PR DESCRIPTION
**Pull request by Alanamosse**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

These are R-packages useful for ecology studies. 
Dependencies for r-gbif and r-spocc, comming soon

### CODE REVIEW

File: recipes/r-wicket/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+requirements:
+  build:
+    - r-base
+    - gcc
+    - r-bh
+    - r-rcpp
+
+  run:
+    - r-base
+    - libgcc         # [win]
```
Comment (jerowe) [inline recipes/r-wicket/meta.yaml]: Could you get rid of the #[win] tag? Bioconda does not build for windows, so its not really applicabble.

Comment (Alanamosse) [reply recipes/r-wicket/meta.yaml]: Ha yes indeed, some skeleton cran comments i forgot to remove

**Comment by Alanamosse** [comment]
@jerowe is it ok for you now ?

**Comment by Alanamosse** [comment]
Hey sorry to ping you one more time @jerowe but can you confirm I can 'Squash and merge' ? 
thank you

**Comment by jerowe** [comment]
@Alanamosse , please ping me when I get distracted. It is the only way! ;-)

Thanks!

**Comment by Alanamosse** [comment]
I will not hesitate next time ! Thks a lot :)

**Pull request merged**